doc/install/install-storage-cluster.rst

```diff
@@ -19,7 +19,9 @@ update APT's database and install Ceph::
 Installing with RPM
 ===================
 
-To install Ceph with RPMs, execute the following steps:
+To install Ceph with RPMs on RHEL 8 and derivatives the next steps are not needed because DNF support repository priorities natively.
+
+On RHEL 7 and derivatives execute the following steps for supporting repository priorities:
```

Comment:
@sandrobonazzola hi Sandro, i think we don't build for RHEL7/CentOS7 anymore. see https://shaman.ceph.com/builds/ceph/master/. could you drop the section mentioning the no-more-needed steps instead?

FWIW, i created #42130 to document the fact that we only build for el8.